### PR TITLE
dropzone: DropzoneOptions url can take callback as well

### PR DESCRIPTION
**Pull request by robob4him**
See: http://www.dropzonejs.com/#config-url

Please fill in this template.

- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `tsc` without errors.
- [N/A] Run `npm run lint package-name` if a `tslint.json` is present.

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: http://www.dropzonejs.com/#config-url
- [?] Increase the version number in the header if appropriate.
- [?] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: dropzone/index.d.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ declare namespace Dropzone {
 	}
 
 	export interface DropzoneOptions {
-		url?: string;
+		url?: string|((files: DropzoneFile[]) => string);;
```
Comment (DanielRosenwasser) [inline dropzone/index.d.ts]: Can you remove the extra semicolon?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: dropzone/index.d.ts (hunk 1 above)
Comment (outring) [inline dropzone/index.d.ts]: And also remove the braces around, they're redundant.

**Comment by outring** [comment]
Hi, @robob4him! Could you also write a test for it? If you need some help with this — tell me.

**Comment by jramsay** [comment]
Thanks for your contribution. This PR has unaddressed comments and cannot be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**